src/database.ts

```diff
@@ -189,4 +189,52 @@ export async function PollTokensFromDatabase(tenantAllias: string) {
         console.error(error);
     });
     return data;
+}
+
+
+
+
+
+/**
+ * Used to store temporary user information
```

Comment:
Should we remove the AddTokensToDatabase function if this will now use the AddTempUser function instead?

Reply:
I don't think it can hurt to have it in at the moment